tests/ui/specs/Popup/Popup-specs.js

```diff
@@ -6,185 +6,186 @@ describe('Popup', function () {
 
 	const popupCommon = Page.popupCommon;
 
-	beforeEach(function () {
-		Page.open();
+	beforeEach(async function () {
+		await Page.open();
 	});
 
-	it('should focus the first button on start', function () {
-		expect(popupCommon.buttonPopup1.isFocused()).to.be.true();
+	it('should focus the first button on start', async function () {
+		expect(await popupCommon.buttonPopup1.isFocused()).to.be.true();
 	});
 
-	it('should not have the popup on start', function () {
-		expectClosed(popupCommon);
+	it('should not have the popup on start', async function () {
+		await expectClosed(popupCommon);
 	});
 
 	describe('Popup with AutoDismiss', function () {
 
 		const popup = Page.components.popup1;
 
-		it('should have correct title', function () {
-			Page.waitTransitionEnd(3000, undefined, () => {
+		it('should have correct title', async function () {
+			await Page.waitTransitionEnd(3000, undefined, async () => {
 				popupCommon.buttonPopup1.click();
 			});
-			expectOpen(popupCommon);
-			validateTitle(popup, 'Popup with AutoDismiss');
+			await expectOpen(popupCommon);
+			await validateTitle(popup, 'Popup with AutoDismiss');
 		});
 
 		describe('5-way', function () {
 
-			it('should spot default button in popup container', function () {
-				Page.waitTransitionEnd(3000, 'popup open', () => {
+			// it('should spot default button in popup container', async function () {
```

Comment:
It will be fixed on WRN-17780.,
Please fix it once 17780 done

Reply:
they work with enact/master. thanks for the tip